### PR TITLE
Add Stack TVL adapter

### PR DESCRIPTION
**Pull request by SeaZarrgh**
##### Name (to be shown on DefiLlama):

Stack

##### List of audit links if any:

https://docs.stackmore.xyz/using-stack/security-and-audits

##### Website Link:

https://www.stackmore.xyz/

##### Logo (High resolution, will be shown with rounded borders):

logo-light

##### Current TVL:

Roughly $695,000 ($585,000 in collateral and $110,000 in staked MORE)

##### Treasury Addresses (if the protocol has treasury)

N/A

##### Chain:

re.al

##### Short Description (to be shown on DefiLlama):

Stack Finance is a borrowing and leverage platform that uses a variety of crypto native and real world asset tokens as collateral to borrow the stablecoin $MORE.

##### Token address and ticker if any:

MORE: 0x25ea98ac87A38142561eA70143fd44c4772A16b6
https://explorer.re.al/token/0x25ea98ac87A38142561eA70143fd44c4772A16b6

##### Category (full list at https://defillama.com/categories) *Please choose only one:

CDP

##### Oracle Provider(s): Specify the oracle(s) used (e.g., Chainlink, Band, API3, TWAP, etc.):

DIA, static price oracles

##### Implementation Details: Briefly describe how the oracle is integrated into your project:

Oracles are used to calculate position LTVs and thus control liquidations.

##### Documentation/Proof: Provide links to documentation or any other resources that verify the oracle's usage:

https://docs.stackmore.xyz/using-stack/asset-valuations-and-oracles
https://docs.stackmore.xyz/using-stack/contracts-and-addresses

##### forkedFrom (Does your project originate from another project):

N/A

##### methodology (what is being counted as tvl, how is tvl being calculated):

TVL counts the collateral tokens that are deposited within the respective Stack vault, as well as the amount of staked MORE.

##### Github org/user (Optional, if your code is open source, we can track activity):

https://github.com/Pearl-Finance/stack

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/stack exports TVL: 
        
 
 
    real                      648.33 k
    
    total                    648.33 k

File: projects/helper/tokenMapping.js [hunk 1]
```diff
@@ -166,10 +166,13 @@ const fixBalancesTokens = {
     '0x2967e7bb9daa5711ac332caf874bd47ef99b3820': { coingeckoId: 'wrapped-stbtc', decimals: 18 },
   },
   real: {
-    [ADDRESSES.null]: { coingeckoId: 'bitcoin', decimals: 18 },
-    '0x90c6e93849e06ec7478ba24522329d14a5954df4': { coingeckoId: 'ethereum', decimals: 18 },
+    [ADDRESSES.null]: { coingeckoId: 'real-ether', decimals: 18 },
```
[the remark this answers is not in the document]
Comment (g1nt0ki) [reply projects/helper/tokenMapping.js:169]: any idea what is the price diff between real-ether & ethereum?

**Comment by llamatester** [comment]
The adapter at projects/stack exports TVL: 
        
 
 
    real                      935.21 k
    
    total                    935.21 k

**Comment by g1nt0ki** [comment]
@SeaZarrgh thanks for the PR

I removed staked MORE from tvl, we dont count staked DAI as part of maker tvl as the tokens backing it is already counted towards tvl

File: projects/helper/tokenMapping.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (SeaZarrgh) [reply projects/helper/tokenMapping.js]: We're in contact with CG, should be fixed soon. real-ether's value is slightly higher as it's staked ETH.

**Pull request merged**